src/game_inventory.cpp

```diff
@@ -522,6 +522,21 @@ class comestible_inventory_preset : public inventory_selector_preset
                 }
             }, _( "JOY" ) );
 
+            append_cell( []( const item_location & loc ) {
+                const int health = loc->is_comestible() ? loc->get_comestible()->healthy : 0;
+                if( health > 0 ) {
+                    return _( "<good>healthy</good>" );
```

Comment:
I would appreciate translation context with this and other translatable lines as they might overlap with other entries.

Reply:
Hi can you explain? I don't understand

Reply:
Different languages may have different words for "healthy" in "the player is healthy" and "this is a healthy thing to eat". This is conveyed in CDDA using the function pgettext - see [doc/TRANSLATING.md#translation-functions](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/TRANSLATING.md#translation-functions). I'd guess in this case one would be using pgettext( "A food/drink", "<good>healthy</good> ), but I haven't explored this deeply.

Reply:
Thanks for the clarification!

Reply:
You can use special format of comments (`//~`) on the line before `_( ... )` to hint translators. Search the codebase for "//~" to see examples.

Reply:
Good point also. As far as I can tell, the context form is preferable because it better enables reuse (only having to do the translation once, in case someone else wants to display info about healthiness - in Magiclysm from a spell, for instance), but I may well be incorrect on this.

Reply:
In this case a context is prefered because comments do not enable separate translation and is used only to convey additional information to the translator.

Reply:
Alright, I couldn't decide between the comments and the pgettext function so..... i added both!
**There is never enough context** 😄